src/widgets-service.js

```diff
@@ -151,7 +158,9 @@ WidgetsService.prototype.createListModel = function (attrs, layer) {
   var widgetModel = new WidgetModel({
     id: attrs.id,
     type: 'list',
-    title: attrs.title
+    title: attrs.title,
+    attrsNames: ['title'],
+    dataviewModelAttrsNames: ['columns', 'columns_title']
```

Comment:
I added it since it was included in the examples, it's not used? I'm more than happy to :fire: it is that's the case